src/lib.rs

```diff
@@ -241,21 +248,26 @@ mod hwb;
 
 mod hues;
 
-mod tristimulus;
 mod convert;
 mod equality;
+pub mod chromatic_adaptation;
+pub mod white_point;
+pub mod matrix;
+pub mod rgb_primaries;
```

Comment:
...or maybe we should just reexport `Mat3` and keep `matrix` private. It doesn't really give anything more useful than that. This isn't a linear algebra library, after all.